src/platform/stm32/libs/libusbhost/src/usbh_lld_stm32f4.c

```diff
@@ -253,6 +253,10 @@ static void write(void *drvdata, const usbh_packet_t *packet)
 	uint32_t dpid;
 	if (packet->endpoint_type == USBH_ENDPOINT_TYPE_CONTROL) {
 		if (packet->control_type == USBH_CONTROL_TYPE_DATA) {
+			// For Status OUT stage, Length==0, Status Out PID = 1 */
+			if( !packet->datalen ) {
```

Comment:
`if (!packet->datalen) {` according to the coding conventions I guess.

Reply:
ok